src/test/test-path-util.c

```diff
@@ -26,6 +26,15 @@
                 assert_se(path_equal(b, a) == !result);   \
         }
 
+#define test_path_kill_slashes_one(in, out, out_dot) {                  \
+                char *p;                                                \
+                                                                        \
+                p = strdupa(in);                                        \
+                assert_se(streq(path_kill_slashes_and_dot(p, false), out)); \
+                p = strdupa(in);                                        \
+                assert_se(streq(path_kill_slashes_and_dot(p, true), out_dot)); \
+        }
```

Comment:
why make this a macro? why not a regular function? (some test cases use macros like this for no real reason i can see, but we should not add more...)

Reply:
Now I use function instead of macro.